onnxscript/tests/function_libs/torch_lib/ops_test_common.py

```diff
@@ -56,6 +56,8 @@
 
 TEST_OPSET_VERSION = 18
 IS_WINDOWS = os.name == "nt"
+# The number of reruns for flaky tests
+FLAKY_RERUNS = 5
```

Comment:
Would 5 be too much?

Reply:
Potentially? But we should be able to handle them with 2-3 reruns; I set to 5 just in case (following the example in the pytest-rerunfailures library). We can adjust this if the tests don't run as we want them to